docs/occurrences.rst

```diff
@@ -3,12 +3,32 @@
 occurrences module
 ==================
 
-.. py:module:: pyobis
-
-.. automethod:: occurrences.search
-.. automethod:: occurrences.get
-.. automethod:: occurrences.grid
-.. automethod:: occurrences.getpoints
-.. automethod:: occurrences.point
-.. automethod:: occurrences.tile
-.. automethod:: occurrences.centroid
+.. py:module:: pyobis.occurrences
+
+.. autoclass:: OBISQueryResult
+
+An OBISQueryResult object for fetching occurrence records.
+
+Usage
+#####
+
+.. code-block:: python
+
+    from pyobis.occurrences import OBISQueryResult as OQR
```

Comment:
While `numpy as np` and others are well established short versions of imports today, with better autocomplete functionality, these short versions kind of obfuscate the code. In our docs I'd rather we use the full name of the class to make it easier to follow the example along.

Reply:
Thank you so much for the review! Should I update those docs/docstrings in another PR?

Reply:
Maybe. Definitely don't change this one. If @7yl4r agrees with my suggestion we can send another PR later.

Reply:
Full package names are my preference, even with packages like `numpy`. But I agree that we save that for later.